src/cleanup_data.py

```diff
@@ -0,0 +1,70 @@
+# author: Tran Doan Khanh Vu
+# date: 2020-11-27
+
+"""Load a csv / feather data file from a local input file and split into test and training data set and write to 2 separate local output files. The output file will be either a csv or a feather file format, which is determined by the extension.
+
+Usage: src/cleanup_data.py --in_file=<in_file> --out_training_file=<out_training_file> --out_test_file=<out_test_file> [--random_state=<random_state>] [--test_size=<test_size>]
```

Comment:
I think this should be edited to the below, correct?
```suggestion
Usage: python src/cleanup_data.py --in_file=<in_file> --out_training_file=<out_training_file> --out_test_file=<out_test_file> [--random_state=<random_state>] [--test_size=<test_size>]
```

Reply:
Thanks @yaz-saleh , I have added that part.